src/server/scripts/Shadowlands/zone_exiles_reach.cpp

```diff
@@ -0,0 +1,812 @@
+                                garrick2 = creature;
+                        }
+                        if (garrick1)
+                            printf("Garrick1 guid %s\r\n", garrick1->GetGUID().ToString().c_str());
+                        if (garrick2)
+                            printf("Garrick2 guid %s\r\n", garrick2->GetGUID().ToString().c_str());
+                        if (!garrick1 || !garrick2)
+                            return;
+                        garrick2->SummonPersonalClone(garrick1->GetPosition(), TempSummonType(TEMPSUMMON_TIMED_DESPAWN), 60s, 0, 0, player);
+                        */
+                    }
+                }
+                break;
+            default:
+                break;
+            }
+            _spawn = false;
+        }
+    }
+private:
+    EventMap _events;
+    bool _spawn;
+public:
+    ObjectGuid _playerGUID;
+    ObjectGuid _garrickGUID;
+};
+
+enum QuestScripts
+{
+    CONVERSATION_WARMING_UP         = 12798,
+    NPC_WARLORD_BREKA_GRIMAXE1      = 166573,
+    NPC_WARLORD_BREKA_GRIMAXE2      = 166824,
+    NPC_ABOAT                       = 156403,
+    NPC_HBOAT                       = 168039,
+    NPC_INVISBUNNY                  = 155125,
+    QUEST_WARMING_UP_ALLIANCE       = 56775,
+    QUEST_WARMING_UP_HORDE          = 59926
+};
+
+class quest_warming_up : public QuestScript
+{
+public:
+    quest_warming_up() : QuestScript("quest_warming_up") { }
+
+    void OnQuestStatusChange(Player* player, Quest const* quest, QuestStatus /*oldStatus*/ , QuestStatus newStatus) override
+    {
+        switch (newStatus)
+        {
+            case QUEST_STATUS_COMPLETE:
+                Conversation::CreateConversation(CONVERSATION_WARMING_UP, player, *player, player->GetGUID(), nullptr);
+                break;
+            case QUEST_STATUS_REWARDED:
+                if (quest->GetQuestId() == QUEST_WARMING_UP_ALLIANCE)
+                {
+                    if (Creature* invisbunny = player->FindNearestCreature(NPC_INVISBUNNY, 30.0f))
+                    {
+                        ENSURE_AI(npc_alliance_boat_invisbunny, invisbunny->GetAI())->_playerGUID = player->GetGUID();
+                        if (Creature* garrick = player->FindNearestCreature(NPC_CAPTAIN_GARRICK, 5.0f))
+                            ENSURE_AI(npc_alliance_boat_invisbunny, invisbunny->GetAI())->_garrickGUID = garrick->GetGUID();
+                        invisbunny->AI()->SetData(1, 1);
+                    }
+                }
+                else if (quest->GetQuestId() == QUEST_WARMING_UP_HORDE)
+                {
+                    if (Creature* breka1 = player->FindNearestCreature(NPC_WARLORD_BREKA_GRIMAXE1, 10.0f))
+                        if (Creature* breka2 = player->FindNearestCreature(NPC_WARLORD_BREKA_GRIMAXE2, 75.0f))
+                            breka2->SummonPersonalClone(breka1->GetPosition(), TempSummonType(TEMPSUMMON_TIMED_DESPAWN), 18s, 0, 0, player);
```

Comment:
(You shouldn't mark a comment as resolved which you didn't resolve)

Remove the 18s timer, the npc is supposed to despawn itself when reaching the destination